src/wp-includes/post.php

```diff
@@ -2215,13 +2215,36 @@ function add_post_type_support( $post_type, $feature, ...$args ) {
 	global $_wp_post_type_features;
 
 	$features = (array) $feature;
+	if ( in_array( 'footnotes', $features, true ) ) {
+		$index = array_search( 'footnotes', $features, true );
+		unset( $features[ $index ] );
+		$features[] = 'footnotes';
+	}
```

Comment:
Hmm, this code will run 2x, as it's also being done in `WP_Post_Type::add_supports()` which first moves it to last and then invokes `add_post_type_support()`. Wondering if it can be combined to do this only once.

Reply:
Thinking more about this ... 

How about moving the "move to last" code in `WP_Post_Type::add_supports()` to here?

What might the impact be within `WP_Post_Type`'s usage of its `supports` property? Hmm